tests/test-logging-monitoring/test-lambda-dashboard-requirements.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+)
+
+type PrometheusSeries struct {
+	Status string              `json:status`
+	Data   []map[string]string `json:data`
+}
+
+func checkMetricsAndlabels(metric string, labels ...string) error {
+	url := prometheusURL + "/api/v1/series"
+	url += "?match[]=" + metric
+
+	for _, l := range labels {
+		u := fmt.Sprintf("%s{%s=~\"..*\"}", url, l)
+		s := PrometheusSeries{}
+
+		resp, err := http.Get(u)
+		if err != nil {
+			return fmt.Errorf("Call to prometheus failed: %v", err)
+		}
+
+		defer resp.Body.Close()
+		bodyBytes, _ := ioutil.ReadAll(resp.Body)
+		json.Unmarshal(bodyBytes, &s)
+
+		if s.Status != "success" {
```

Comment:
may be printing the response code  also would be useful.

Reply:
fixed